lib/userConf.js

```diff
@@ -69,11 +69,34 @@ const getConf = async function getConf(key, envVar, ...defaultValue) {
   return userConf[key];
 };
 
+// Matches printable ASCII-only values.
+const ascii = /^[\u0020-\u007E]*$/;
+
+// Validates a header can be sent.  In practice ISO 8859-1 is allowed,
+// but that is rejected by the Binaris backend.  Disallow it on
+// principle, for aesthetic reasons
+const validateHeaderValue = async function validateHeaderValue(value, context) {
+  if (!ascii.test(value)) {
+    throw new Error(`Non-ASCII characters are not allowed in ${context}.`);
```

Comment:
afaict all other errors do not end in a comma.
i don't have strong preference either way, but i do strongly prefer consistency.
please make it consistent

Reply:
Thanks; done!